app/views/content/how-to-apply/_questions.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="category__questions__container">
+
+  <section class="container category__questions">
+
+    <div class="category__question">
+      <h2>Want to chat with an adviser?</h2>
+
+      <p>
+        Get free, one-to-one support with your application from an adviser with years of teaching experience.
+        Chat as little or often as you like by text, phone or email.
+      </p>
+
+      <a href="/teacher-training-adviser/sign_up/identity" class="button">Get an adviser</a>
+    </div>
+
+    <div class="category__question">
+      <h2>Want to get face-to-face advice?</h2>
+
+      <p>
+        Attend a Get Into Teaching event, either online or in person, to get tips on your application.
+        Talk to training providers and meet others considering a career in teaching.
```

Comment:
Not sure whether `meet others....` is so important here (but please disagree!). I wonder whether it is more something like 

`Meet training providers to find out about courses in your area and talk to teachers about what teaching is really like.`

(That's not very well written!)